esmvaltool/cmorizers/data/downloaders/datasets/nsidc_0116_nh.py

```diff
@@ -0,0 +1,47 @@
+"""Script to download NSIDC-0116-nh."""
+
+from datetime import datetime
+
+from dateutil import relativedelta
+
+from esmvaltool.cmorizers.data.downloaders.wget import NASADownloader
+
+
+def download_dataset(config, dataset, dataset_info, start_date, end_date,
+                     overwrite):
+    """Download dataset.
+
+    Parameters
+    ----------
+    config : dict
+        ESMValTool's user configuration
+    dataset : str
+        Name of the dataset
+    start_date : datetime
+        Start of the interval to download
+    end_date : datetime
+        End of the interval to download
+    overwrite : bool
+        Overwrite already downloaded files
+    """
+    if not start_date:
+        start_date = datetime(1997, 1, 1)
+    if not end_date:
+        end_date = datetime(2018, 1, 1)
+    loop_date = start_date
```

Comment:
Unless there is a reason, could you use the same dates as for the "south data"? This is 1979-2020 and would be more consistent with the obs data pool on Mistrla.

Reply:
It was probably a copy paste mistake. Fixed now